.pre-commit-config.yaml

```diff
@@ -15,3 +15,8 @@ repos:
         stages: [commit-msg]
     repo: https://github.com/jorisroovers/gitlint
     rev: v0.18.0
+  - hooks:
+      - id: go-mod-tidy
+        args: [--verbose]
```

Comment:
Does this has any effect?

Reply:
apparently not, I removed it

Reply:
The important bit is whether it makes a difference when there's an error. I think you should try to fail the check again and see if it makes a difference in that case.

Reply:
I failed the check a few times and it was the same with and without it